openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1352,9 +1352,10 @@ This specification defines the following Credential Issuer Metadata parameters:
-  * `cryptographic_suites_supported`: OPTIONAL. Array of case sensitive strings that identify the cryptographic suites that are supported for the `cryptographic_binding_methods_supported`. Cryptographic algorithms for Credentials in `jwt_vc` format should use algorithm names defined in [@IANA.JOSE.ALGS]. Cryptographic algorithms for Credentials in `ldp_vc` format should use signature suite names defined in [@LD_Suite_Registry].
-  * `proof_types`: OPTIONAL. Array of case sensitive strings, each representing a `proof_type` that the Credential Issuer supports, as defined in (#credential_request), one of which MUST be used in the Credential Request. If this array is non-empty and present, the Credential Issuer requires proof of possession of the cryptographic key material. If the parameter is omitted or the array is empty, the Credential Issuer does not require proof of possession of the cryptographic key material.
+  * `cryptographic_binding_methods_supported`: OPTIONAL. Array of case sensitive strings that identify the representation of the cryptographic key material that the issued Credential is bound to, as defined in (#credential-binding). Support for keys in JWK format [@!RFC7517] is indicated by the value `jwk`. Support for keys expressed as a COSE Key object [@!RFC8152] (for example, used in [@!ISO.18013-5]) is indicated by the value `cose_key`. When the Cryptographic Binding Method is a DID, valid values are a `did:` prefix followed by a method-name using a syntax as defined in Section 3.1 of [@!DID-Core], but without a `:`and method-specific-id. For example, support for the DID method with a method-name "example" would be represented by `did:example`. Support for all DID methods listed in Section 13 of [@DID_Specification_Registries] is indicated by sending a DID without any method-name.
+  * `credential_signing_alg_values_supported`: OPTIONAL. Array of case sensitive strings that identify the algorithms that the Issuer uses to sign the issued Credential. Algorithm names used are determined by the Credential format and are defined in (#format_profiles).
+  * `proofs_supported`: OPTIONAL. Object that describes specifics of the key proof(s) that the Credential Issuer supports. This object contains a list of name/value pairs, where each name is a unique identifier of the supported proof type being described. This identifier is used in the Credential Request defined in (#credential_request) to communicate to the Wallet specifics about the proof of possession of the cryptographic key material. A list of valid key proof type values defined by this specification can be found in (#proof_types). The value in the name/value pair is an object that contains metadata about the key proof and contains the following parameters defined by this specification:
```

Comment:
why `proofs_supported` and not just keep `proof_types`?

Reply:
The object includes not just information about the proof type but also the algs so more general name seemed more accurate.

Reply:
Aren't these signature algs nevertheless applicable for a certain type of proofs? I mean, that's why we moved them underneath the proof_type.

Reply:
that is true. Would `proof_types_supported` work?

Reply:
I'm fine with `proof_types_supported`

Reply:
WG call on 31st Of Jan (NZ): consensus for `proof_types_supported`. Additionally, section 7.2.1's title is changed to "proof types".